test/release/examples/primers/recordio.chpl

```diff
@@ -114,7 +118,7 @@ proc MyRecord.init(i: int = 0, r: real = 0.0, s: string = "") {
 
 proc MyRecord.init(r: fileReader(?)) throws {
   this.init();
-  readThis(r);
+  deserialize(r, r.deserializer);
```

Comment:
I'm pretty sure the deserializing initializer signature is `init(reader: fileReader(?), ref deserializer: ?dt) throws`. 

(With `initDeserializable` implemented, I believe you can replace the `writeSerializable, readDeserializable` interface markers on the record with just: `serializable`).

Reply:
Good call, I was assuming prior adjustments on that front were complete.  Thanks!